wdl-lint/tests/lints/shellcheck-ok/source.wdl

```diff
@@ -0,0 +1,79 @@
+#@ except: DescriptionMissing, RuntimeSectionKeys, MatchingParameterMeta, NoCurlyCommands
+
+## This is a test of having no shellcheck lints
+
+version 1.1
+
+task test1 {
+    meta {}
+
+    parameter_meta {}
+
+    input {
+      Boolean i_quote_my_shellvars
+      Int placeholder
+    }
+
+    command <<<
+      set -eo pipefail
+
+      echo "$placeholder"
+
+      if [[ "$i_quote_my_shellvars" ]]; then
+        echo "shellcheck will be happy"
+      fi
+    >>>
+
+    output {}
+
+    runtime {}
+}
+
+task test2 {
+    meta {}
+
+    parameter_meta {}
+
+    input {
+      Int placeholder
+    }
+
+    command {
+      set -eo pipefail
+
+      echo "$placeholder"
+
+      if [[ "$I_quote_my_shellvars" ]]; then
+        echo "all is well"
+      fi
+    }
+
+    output {}
+
+    runtime {}
+}
+
+task test3 {
```

Comment:
Can we add a `test4` that uses the shellcheck suppression comment _within_ the command block? e.g.
```
command <<<
    #shellcheck-disable=SCfoo
    if [[ $im_a_bad_example ]]; then
       echo bar
   fi
>>>
```